setup.py

```diff
@@ -60,7 +60,7 @@
     long_description_content_type="text/x-rst",
     author="Adafruit Industries",
     author_email="circuitpython@adafruit.com",
-    python_requires=">=3.7.0",
+    python_requires=">=3.9.0",
```

Comment:
I'd like to keep this at 3.8.0 so that we don't leave the NVIDIA jetson folks behind.